crates/camera/src/camera.rs

```diff
@@ -23,13 +23,13 @@ const CAMERA_HORIZONTAL_SPEED: InverseSecond = Quantity::new_unchecked(2.0);
 /// Minimum camera distance from terrain achievable with zooming along.
 const MIN_CAMERA_DISTANCE: Metre = Quantity::new_unchecked(20.);
 /// Maximum camera distance from terrain achievable with zooming alone.
-const MAX_CAMERA_DISTANCE: Metre = Quantity::new_unchecked(100.);
+const MAX_CAMERA_DISTANCE: Metre = Quantity::new_unchecked(300.);
```

Comment:
You are right, at this point in time this change makes sense.

A bit of a context:

* In the long run, I would like to implement a custom LOD so zooming out a lot is feasible from performance perspective.

  * Until this happens, it might be necessary to change this back to a lower value after we plug in better unit models (with possibly higher triangle counts etc.).

* The appearance / UI has to change on high camera distances: drawing simple colored squares instead of true 3D models, not displaying health bars and selection circles etc. More info here #153.

* Implementing some kind of a mini map (displaying at least all units & buildings & camera) is high on my priority list. That would solve most of the issue you mention (long scroll -> single click on the mini map, getting disoriented -> looking where you are looking at at the mini map).

Reply:
This is related #68.